src/Plugin/Field/FieldWidget/ModerationStateWidget.php

```diff
@@ -28,16 +28,27 @@ class ModerationStateWidget extends CoreModerationStateWidget {
    */
   public function formElement(FieldItemListInterface $items, $delta, array $element, array &$form, FormStateInterface $form_state) {
     /** @var \Drupal\Core\Entity\ContentEntityInterface $entity */
-    $entity = $items->getEntity();
+    $entity = $original_entity = $items->getEntity();
```

Comment:
I think "original_entity" is a bit misleading. The name suggests, that it keeps being unchanged. But instead, it is actually the one that gets changed

Reply:
I used $original_entity, because that's the name that is used in the base class as well.

Reply:
Ok